api/verify.go

```diff
@@ -200,7 +200,7 @@ func (a *API) recoverVerify(ctx context.Context, conn *storage.Connection, user
 			return terr
 		}
 		if !user.IsConfirmed() {
-			if terr = models.NewAuditLogEntry(tx, instanceID, user, models.UserSignedUpAction, nil); terr != nil {
+			if terr = models.NewAuditLogEntry(tx, instanceID, user, models.LoginAction, nil); terr != nil {
```

Comment:
Wouldn't the user signed up action be correct since it's guarded by the `!user.IsConfirmed()` check? Maybe a separate audit log entry should be made with the login action if the user is confirmed?

Reply:
hmm good point, yeah i overlooked this, let me make a change! thanks for catching this 🔥